cli/command/stack/opts.go

```diff
@@ -9,13 +9,8 @@ import (
 	"github.com/spf13/pflag"
 )
 
-func addComposefileFlag(opt *string, flags *pflag.FlagSet) {
-	flags.StringVarP(opt, "compose-file", "c", "", "Path to a Compose file")
-}
-
-func addBundlefileFlag(opt *string, flags *pflag.FlagSet) {
-	flags.StringVar(opt, "bundle-file", "", "Path to a Distributed Application Bundle file")
-	flags.SetAnnotation("bundle-file", "experimental", nil)
+func addFileFlag(opt *string, flags *pflag.FlagSet) {
+	flags.StringVarP(opt, "file", "f", "", "Path to a Compose or a Distributed Application Bundle file")
```

Comment:
Also, we should discuss the shorthand `-f`; there was talk of a possible `--force` option (for certain changes, such as changing mode to "global" for a service

/cc @dnephin